spec/services/select_outdated_uploads_strategy_spec.rb

```diff
@@ -35,24 +35,19 @@
     end
   end
 
-  # rubocop:disable RSpec/LetSetup
   describe '#select' do
     subject(:selection) { strategy.select }
 
-    let(:old_days_ago1) { 28.days.ago }
-    let(:old_days_ago2) { 14.days.ago }
-    let!(:outdated_upload1) { create(:active_storage_blob, created_at: old_days_ago1) }
-    let!(:outdated_upload2) { create(:active_storage_blob, created_at: old_days_ago2) }
-    let!(:current_upload1) { create(:active_storage_blob) }
-    let!(:current_upload2) { create(:active_storage_blob) }
-    let!(:current_upload3) { create(:active_storage_blob) }
```

Comment:
Were these omitted or are they not needed?

Reply:
I didn't see where they were used.